flows/exam-1.yml

```diff
@@ -28,24 +28,28 @@ rules:
         permissions: []
 
     -
-        if_in_facility: test_center
-        if_session_duration_shorter_than_minutes: 0.5
-        permissions: [view, submit_answer, end_session, see_session_time, lock_down_as_exam_session]
+        if_has_role: [instructor]
+        permissions: [view, submit_answer, change_answer, end_session, see_correctness, see_session_time]
 
     -
-        if_in_facility: test_center
-        message: "You exam will end soon."
-        if_session_duration_shorter_than_minutes: 1
-        permissions: [view, submit_answer, end_session, see_session_time, lock_down_as_exam_session]
+        if_in_facility: "cbtf"
```

Comment:
- "CBTF" again is UIUC-specific terminology.

Reply:
- Nothing wrong with *adding* this in a comment, but the generic `test_center` is probably better.
- It would probably also be good to explain that these facility names are configured per-instance in `local_settings.py`.